website/src/pages/tutorial/basic/04-a-simple-query.mdx

```diff
@@ -124,8 +124,6 @@ Awesome, the server responds with the data you defined in `links`:
 }
 ```
 
-![data returned from the server](/assets/tutorial/basic/04-a-simple-query/feed-query-response.png)
```

Comment:
This image is actually a duplicate of /assets/tutorial/basic/05-a-simple-mutation/post-link-mutation-response.png.

The screen shot references on line 97 needs to be updated to only show the one feed link that the hardcoded `links` variable has. However, the screen shot on line 97 references the data returned after a new feed link is added.